### DISCUSSION TITLE
How to add emulated games?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#475] [Q&A] [answered]
I wanna add some Wii U Games to open them with Cemu.
I can't try it,cause I don't have any Wii U Roms on my pc right now.

**Comment by tkashkin** [comment, marked as answer]
1. Configure custom emulators in Settings and set game file patterns for them.
2. Click the `Add game` button in the main window and open `Import emulated games` dialog.
3. Select a directory to scan and GameHub should find all matching files.
4. Check the files you want to import, select the emulator to run them if multiple emulators or RetroArch cores can handle that file, and optionally change the filenames before importing.
5. You can also use `Download game images` from the `Add game` menu to automatically find images for games that don't have them.

**Comment by aasifkft2** [comment]
Gta 5
